workers/generic-worker/artifactfeature.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/taskcluster/taskcluster/v83/internal/scopes"
+	"github.com/taskcluster/taskcluster/v83/workers/generic-worker/artifacts"
+)
+
+type (
+	ArtifactFeature struct {
+	}
+
+	ArtifactTaskFeature struct {
+		task *TaskRun
+	}
+)
+
+func (af *ArtifactFeature) Name() string {
+	return "Artifact Feature"
```

Comment:
Maybe we could switch up the name on this so the logs don't read a little weird, currently you'd see `Starting task feature Artifact Feature...` and `Stopping task feature Artifact Feature...`. No other features have the word `Feature` in the name, too.

```suggestion
	return "Artifact Uploads"
```